apps/frontend/src/app/PageArtifact/ArtifactEditor/Components/SubstatInput.tsx

```diff
@@ -37,6 +38,8 @@ export default function SubstatInput({ index, artifact, setSubstat }: { index: n
   if (!rollNum && key && value) error = error || t`editor.substat.error.noCalc`
   if (allowedRolls < 0) error = error || t("editor.substat.error.noOverRoll", { value: allowedRolls + rollNum })
 
+  const marks = useMemo(() => key ? artifactSubRolls(rarity, key).map(v => ({ value: v, })) : [{ value: 0 }], [key, rarity])

```

Comment:
You can't drag the slider back to 0 once you've moved it. Feels kind of unintuitive if you touch the slider but want to move it back so you can use the buttons to change the first roll

Reply:
yeah I want to discourage this behavour, since if they leave it at 0 it will actually get validated for min value when they save. they can always enter 0 in the input to go to back to that state